first/style.css

```diff
@@ -0,0 +1,234 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;500&display=swap'); /* font-family: 'Rubik', sans-serif */
+
+body {
+    font-family: 'Rubik', sans-serif;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.header{
+    height:600px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-image: url(./assets/images/5.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+}
+
+.navBar {
+    position: fixed;
+    z-index: 3;
+    top: 0px;
+    display: flex;
+    flex-direction: row;
```

Comment:
recheck this

Reply:
the default is row, why did you keep it? @yousefQJ19

Reply:
i thought it was because the aligning of the elements